### ISSUE TITLE
[4.0.0-beta.0] Ionic datetime component date picker not showing correct year

### ISSUE THREAD

**Issue by mbaljeetsingh** [ionic-team/ionic#15221]
# Bug Report

**Ionic Info**
Run `ionic info` from a terminal/cmd prompt and paste the output below.

![image](https://user-images.githubusercontent.com/872762/44280684-0db98180-a273-11e8-80d0-e586953b8826.png)

**Describe the Bug**
I'm using Ionic datetime component. It displays the correct date that is defined in the typescript file. But when I click on it and the picker opens the year is not right.

Here is the output,
![chrome_2018-08-17_23-05-38](https://user-images.githubusercontent.com/872762/44280845-83bde880-a273-11e8-8d2d-614baafbeb29.png)

On the top right you can see, 17-aug-2018, but when the picker opens the year is 2017.

**Related Code**
```
<ion-datetime display-format="DD-MMM-YYYY" picker-format="DD-MMM-YYYY" (ionChange)="selectedDateChange($event)" [(ngModel)]="selectedDate"></ion-datetime>
```
In the ts file,
```
selectedDate = new Date().toISOString(); 
```

**Expected Behavior**
It should select the correct date when selecting picker.

**Comment by meta0102** [comment]
This issue still exist in beta.3

**Comment by MrSparklle** [comment]
I create a sample project to showed the issue and help to solve:

https://github.com/MrSparklle/Inc4

**Comment by brandyscarney** [comment]
Thanks for the issue! This will be fixed in the `4.0.0-beta.6` release by the following commit: https://github.com/ionic-team/ionic/commit/f30ae889c08677860b4f8d5eda0b97f86e893eb7

Please note that since you are using Angular the properties should be camelCase so instead of `display-format` it should be `displayFormat` and `picker-format` should be `pickerFormat`.
